src/snowflake/snowpark/_internal/utils.py

```diff
@@ -49,7 +49,9 @@
     try:
         from snowflake.connector.cursor import ResultMetadataV2
     except ImportError:
-        ResultMetadataV2 = ResultMetadata
+        from snowflake.connector.cursor import (  # type: ignore
```

Comment:
Can we just not import here? Looks like `ResultMetadataV2` is only used for type checking.